client/js/screens/Campaigns/styles.js

```diff
@@ -1,3 +1,88 @@
 import {StyleSheet} from 'react-native';
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  xxxIcon: {
+    resizeMode: 'contain',
+    height: 30,
+    alignSelf: 'flex-end',
+  },
+  categoryImage: {
+    height: 60,
+    width: 100,
+    resizeMode: 'contain',
+    // alignSelf: 'flex-start',
```

Comment:
remove commented code.

Reply:
Done

Reply:
Please remove the comment. It's still there.